### PR TITLE
Sane numpy

### PR DESCRIPTION
**Pull request by kain88-de**
use the new numpy configurtion for builds

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,21 +12,23 @@ source:
...
   skip: True  # [py3k or win]
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - python
     - setuptools
-    - numpy x.x
+    - numpy 1.8.*  # [not py36]
+    - numpy 1.11.*  # [py36]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: The np111 limitation on py36 is only on Windows. You can use 1.8 everywhere here.

Comment (kain88-de) [inline recipe/meta.yaml]: Ok. But that doesn't explain why the build suddenly complains about a missing dependency.

Comment (ocefpaf) [inline recipe/meta.yaml]: Nope. I'll check on that tomorrow. (Time to clock out here...)

Comment (ocefpaf) [inline recipe/meta.yaml]: Found it before leaving for the day. It seems that the minimum version of numpy forMDAnalysis is 1.10.4.

Comment (kain88-de) [inline recipe/meta.yaml]: That is right. Should have remembered that, since I set that version as a minimum requirement.

**Pull request merged**